app/controllers/assignments_controller.rb

```diff
@@ -252,6 +255,12 @@ def create
 
   def summary
     @assignment = record
+    @lti_deployments = LtiLineItem.where(assessment_id: @assignment.id)
```

Comment:
1. Please put move this into the `@assignment.summary_json` method so that it's included in the JSON payload rather than passed in as props to the table component.
2. Also only send this information when the current role is an instructor.